src/test/resources/application-test.yml

```diff
@@ -1,3 +1,29 @@
+spring:
+
+  datasource:
+    url: 'jdbc:h2:mem:release-dates-db;MODE=PostgreSQL;INIT=create domain if not exists jsonb as json'
```

Comment:
looks like  copy/paste carry over in this file. Can we change

- release-dates-db
- and create_vary